### PR TITLE
Consolidate plugins in build files

### PR DESCRIPTION
**Pull request by dexamundsen**

### CODE REVIEW

File: cli/build.gradle [hunk 1]
```diff
@@ -1,7 +1,4 @@
 plugins {
-    id 'java'
```
Comment (dexamundsen) [inline cli/build.gradle:2]: part of tanagra.java-conventions

File: indexer/build.gradle [hunk 2]
```diff
@@ -1,8 +1,5 @@
 plugins {
     id 'application'
-    id 'java'
```
Comment (dexamundsen) [inline indexer/build.gradle:3]: part of tanagra.java-conventions

File: service/build.gradle [hunk 3]
```diff
@@ -1,10 +1,6 @@
 import org.springframework.boot.gradle.plugin.SpringBootPlugin
 
 plugins {
-    id 'java'
```
Comment (dexamundsen) [inline service/build.gradle:4]: part of tanagra.java-conventions

File: underlay/build.gradle [hunk 4]
```diff
@@ -1,8 +1,4 @@
 plugins {
-    id 'java'
```
Comment (dexamundsen) [inline underlay/build.gradle:2]: part of tanagra.java-conventions

**Review by marikomedlock — APPROVED** [review]

**Pull request merged**